smartmin/views.py

```diff
@@ -141,10 +141,6 @@ def lookup_obj_attribute(self, obj, field):
         # next up is the object itself
         obj_field = getattr(obj, curr_field, None)
 
-        # if it is callable, do so
-        if obj_field and getattr(obj_field, '__call__', None):
```

Comment:
Definitely something we used to use a ton when making dashboards and the like, don't know if we have any uses in RapidPro as the minute, but it sure is useful.

Here's what the Django admin allows:
https://docs.djangoproject.com/en/1.11/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_display

I don't think we need the second option, but 1, 3, 4 seems like reasonable options to have.